tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "ES2021",
-    "lib": ["ES2021"],
```

Comment:
This is the reason, I don't what to update node-fetch to the newest version. 

Probably gonna looks for some other implementation of fetch API.

Reply:
We could use Axios or wait for Node's core fetch implementation as alternatives. Happy to help if needed 😊

Reply:
@antongolub should we add dom? And make zx v8 with node's fetch?